diff --git a/usbSerialExamples/build.gradle b/usbSerialExamples/build.gradle
index 7fdae7e..56cfb90 100644
--- a/usbSerialExamples/build.gradle
+++ b/usbSerialExamples/build.gradle
@@ -15,6 +15,7 @@ android {
 
         testInstrumentationRunner "android.test.InstrumentationTestRunner"
         missingDimensionStrategy 'device', 'anyDevice'
+        vectorDrawables.useSupportLibrary = true
     }
 
     buildTypes {
diff --git a/usbSerialForAndroid/src/main/java/com/hoho/android/usbserial/driver/FtdiSerialDriver.java b/usbSerialForAndroid/src/main/java/com/hoho/android/usbserial/driver/FtdiSerialDriver.java
index 14d9ef5..7e581fd 100644
--- a/usbSerialForAndroid/src/main/java/com/hoho/android/usbserial/driver/FtdiSerialDriver.java
+++ b/usbSerialForAndroid/src/main/java/com/hoho/android/usbserial/driver/FtdiSerialDriver.java
@@ -117,7 +117,7 @@ public class FtdiSerialDriver implements UsbSerialDriver {
         return mPorts;
     }
 
-    private class FtdiSerialPort extends CommonUsbSerialPort {
+    public class FtdiSerialPort extends CommonUsbSerialPort {
 
         public static final int USB_TYPE_STANDARD = 0x00 << 5;
         public static final int USB_TYPE_CLASS = 0x00 << 5;
